python/tvm/relay/expr.py

```diff
@@ -98,39 +125,87 @@ def __init__(self,
 
 @register_relay_node
 class Call(Expr):
-    """A function call in Relay, see tvm/relay/expr.h for more details."""
+    """Function call node in Relay.
+
+    Call node corresponds the operator application node
+    in computational graph terminology.
+
+    Parameters
+    ----------
+    op: tvm.relay.Op or any tvm.relay.Expr with function type.
+        The operation to be called.
 
-    def __init__(self, op, args, attrs, ty_args=None):
-        if not ty_args:
-            ty_args = []
+    args: list of tvm.relay.Expr
+        The arguments to the call.
 
+    attrs: tvm.Attrs, optional
```

Comment:
`attrs: Optional[tvm.Attrs]`

Reply:
is Optional a python type annotation? The current convention is mainly to indicate that the function parameter have default value, not that the field is an optional type

Reply:
yes Optional is part of the typing module in Python. if the function parameter in python has a default value it should probably be a named argument instead, but it's type is still Optional[type].